contentcuration/contentcuration/viewsets/user.py

```diff
@@ -266,7 +272,30 @@ def create_from_changes(self, changes):
 
     def delete_from_changes(self, changes):
         return self._handle_relationship_changes(changes)
-
+    @action(detail=True, methods=['delete'])
+    def remove_self(self, request, pk=None):
+        """
+        Allows a user to remove themselves from a channel as a viewer.
+        """
+        user = self.get_object()
+        channel_id = request.query_params.get('channel_id', None)
+
+        if not channel_id:
+            return HttpResponseBadRequest(_, 'Channel ID is required.')
```

Comment:
I think just `return HttpResponseBadRequest('Channel ID is required.')` is sufficient